services/ext/api.go

```diff
@@ -1490,6 +1490,10 @@ func (api *PublicAPI) ToggleUseMailservers(value bool) error {
 	return api.service.messenger.ToggleUseMailservers(value)
 }
 
+func (api *PublicAPI) TogglePeerSyncing(value bool) error {
```

Comment:
last thing, if you could use  `protocol/requests/toggle_peer_syncing.go`  for this,l similarly as we do for other